dftimewolf/lib/exporters/timesketch.py

```
@@ -58,6 +64,11 @@ def export(self):
       String containing a timesketch URL
     """
     self.output = []
+    # This is not the best way of catching errors, but timesketch_utils will be
+    # deprecated soon.
+    # TODO: Consider using the official Timesketch python API.
+    if not self.timesketch_api.session:
+      return -1
```

-1 isn't a string. Do you want to throw an exception here, instead?